api/google-search/src/main/resources/application.properties

```diff
@@ -1,3 +1,4 @@
 springdoc.swagger-ui.disable-swagger-default-url=true
 springdoc.api-docs.path=/v3/api-docs
 springdoc.swagger-ui.path=/docs
+server.servlet.context-path=/api
```

Comment:
Did you check if Swagger endpoint is working?

Reply:
IF i hit `http://localhost:8080/api/docs`. its working

![image](https://user-images.githubusercontent.com/1652629/103203649-5c41b500-4930-11eb-96a5-72911016a1ba.png)